### PR TITLE
Improve the Admin Machine page of our docs

### PR DESCRIPTION
**Pull request by ahreehong**
*Issue #, if available:*

*Description of changes:*
Improved the admin machine section based on our docs review

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/install/_index.md [hunk 1]
```diff
@@ -32,9 +29,11 @@ This will let you create a cluster in multiple providers for local development o
 - `yq` ([Install](https://github.com/mikefarah/yq/#install))
 
 
-If you are using Ubuntu, use the Docker CE installation instructions to install Docker and not the Snap installation, as described [here.](https://docs.docker.com/engine/install/ubuntu/)
+If you are using Ubuntu, use the Docker CE installation instructions to install Docker as described [here.](https://docs.docker.com/engine/install/ubuntu/)
 
-* For EKS Anywhere Bare Metal, Docker Desktop is not supported
+{{% alert title="Warning" color="warning" %}}
+For EKS Anywhere Bare Metal, Docker Desktop is not supported
```
Comment (mitalipaygude) [inline docs/content/en/docs/getting-started/install/_index.md:35]:
```suggestion
For EKS Anywhere Bare Metal, Docker Desktop is not supported.
```

File: docs/content/en/docs/getting-started/install/_index.md [hunk 2]
```diff
@@ -106,59 +105,7 @@ eksctl anywhere version
...
-
-### For Bare Metal (Tinkerbell)
-You will need to have hookOS and its OS artifacts downloaded and served locally from an HTTP file server.
-You will also need to modify the [hookImagesURLPath]({{< relref "../baremetal/bare-spec/#hookimagesurlpath" >}}) and the [osImageURL]({{< relref "../baremetal/bare-spec/#osimageurl" >}}) in the cluster configuration files.
-Ensure that structure of the files is set up as described in [hookImagesURLPath.]({{< relref "../baremetal/bare-spec/#hookimagesurlpath" >}})
-
-### For vSphere
-If you are using the vSphere provider, be sure that the requirements in the
-[Prerequisite checklist]({{< relref "../vsphere/vsphere-prereq" >}}) have been met.
+For more information on how to prepare the Administrative machine for airgapped environments, go to the [Airgrapped](/docs/getting-started/airgapped/) page.
```
Comment (mitalipaygude) [inline docs/content/en/docs/getting-started/install/_index.md:108]:
```suggestion
For more information on how to prepare the Administrative machine for airgapped environments, go to the [Airgapped](/docs/getting-started/airgapped/) page. 
```

File: docs/content/en/docs/getting-started/install/_index.md [hunk 3]
```diff
@@ -8,13 +8,10 @@ description: >
 EKS Anywhere will create and manage Kubernetes clusters on multiple providers.
 Currently we support creating development clusters locally using Docker and production clusters from providers listed on the [providers]({{< relref "/docs/getting-started/chooseprovider/" >}}) page.
 
-Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run Docker and add some binaries.
-From there, you create the cluster for your chosen provider.
+Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run Docker and add prerequisite utilities. From here you will need to install [`eksctl`](https://eksctl.io) and the [`eksctl-anywhere`](/docs/reference/eksctl/anywhere/) plugin on your Administrative machine.
```
Comment (TerryHowe) [inline docs/content/en/docs/getting-started/install/_index.md:11]:
```suggestion
Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run Docker and add prerequisite utilities.
From here you will need to install [`eksctl`](https://eksctl.io) and the [`eksctl-anywhere`](/docs/reference/eksctl/anywhere/) plugin on your Administrative machine.
```

https://sembr.org/

File: docs/content/en/docs/getting-started/install/_index.md [hunk 4]
```diff
@@ -8,13 +8,10 @@ description: >
...
+You can then create a cluster for your chosen provider.
```
Comment (TerryHowe) [inline docs/content/en/docs/getting-started/install/_index.md:12]:
```suggestion
You can then create a cluster for your chosen provider.
```

**Comment by ahreehong** [comment]
changes were addressed in https://github.com/aws/eks-anywhere/pull/6536

**Pull request closed without merging**